website/docs/r/lb_listener_rule.html.markdown

```diff
@@ -220,10 +240,50 @@ Authentication Request Extra Params Blocks (for `authentication_request_extra_pa
 * `key` - (Required) The key of query parameter
 * `value` - (Required) The value of query parameter
 
+### Condition Blocks
+
+One of more condition blocks can be set per rule. Most condition types can only be specified once per rule except for `http-header` and `query-string` which can be specified multiple times.
+
 Condition Blocks (for `condition`) support the following:
 
-* `field` - (Required) The name of the field. Must be one of `path-pattern` for path based routing or `host-header` for host based routing.
-* `values` - (Required) The path patterns to match. A maximum of 1 can be defined.
+* `field` - (Required) The type of condition. Valid values are `host-header`, `http-header`, `http-request-method`, `path-pattern`, `query-string` and `source-ip`.
+* `host_header` - (Optional) Host headers to match. Host Header block fields documented below. Required if `field` is `host-header` and `values` is empty.
+* `http_header` - (Optional) HTTP headers to match. HTTP Header block fields documented below. Required if `field` is `http-header`.
+* `http_request_method` - (Optional) HTTP request methods to match. HTTP Request Method block fields documented below. Required if `field` is `http-request-method`.
+* `path_pattern` - (Optional) Path patterns to match. Path Pattern block fields documented below. Required if `field` is `path-pattern` and `values` is empty.
+* `query_string` - (Optional) Query strings to match. Query String block fields documented below. Required if `field` is `query-string`.
+* `source_ip` - (Optional) Source IPs to match. Source IP block fields documented below. Required if `field` is `source-ip`.
+* `values` - (Optional, **DEPRECATED**) List of exactly one pattern to match. Only valid when `field` is `host-header` or `path-pattern`, and the `host_header` and `path_pattern` blocks have not been set.
+
+Host Header Blocks (for `host_header`) support the following:
```

Comment:
I would perhaps make every block a h4 as in:
```suggestion
#### Host Header Blocks

Host header blocks (for `host_header`) support the following:
```

So that we can reference the anchor on line 250 for better linking. Thoughts?

Reply:
I agree, something needs to be done with this documentation. I was trying to follow the existing pattern in this document. I'll try with h4.

Reply:
Done.